### PR TITLE
create Card component and integrate it in WhyUsCards and Service components

### PR DESCRIPTION
**Pull request by harshitverma2103**

### CODE REVIEW

File: src/components/Card/index.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { SERVICES_PAGE_PATH } from "../../constants.js";
+import "./style.css";
+
+const Card = ({ icon, title, content, url }) => {
+  const navigate = useNavigate();
+
+  const handleOnClick = () => {
```
Comment (vprince001) [inline src/components/Card/index.jsx:9]: this onClick handler should be passed from outside.

File: src/components/Card/index.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          {icon && <img src={icon} alt="icon" className="card-icon" />}
+        </div>
+        <div className="card-title">
+          <h3>{title}</h3>
+        </div>
+      </div>
+      <div className="card-content">
+        <p>{content}</p>
+      </div>
+      <div className="card-url" onClick={handleOnClick}>
```
Comment (vprince001) [inline src/components/Card/index.jsx:26]: add a condition to show link only when url is provided.

File: src/components/WhyUsCards/index.jsx [hunk 3]
```diff
@@ -1,50 +1,23 @@
 import React from "react";
+import Card from "../Card";
+import { data } from "./whyUs-data.js";
```
Comment (vprince001) [inline src/components/WhyUsCards/index.jsx:3]: fix this file naming

File: src/components/WhyUsCards/whyUs-data.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import group from "../../assets/icons/group.png";
+import mortarboard from "../../assets/icons/mortarboard.png";
+import tech from "../../assets/icons/coding.png";
```
Comment (vprince001) [inline src/components/WhyUsCards/whyUs-data.js:3]: Name these icons as group-icon, mortarboard-icon and tech-icon

File: src/pages/Services/Cards/index.jsx [hunk 5]
```diff
@@ -1,24 +1,9 @@
 import React, {useEffect, useState} from "react";
 import {scrollToElement} from "../../../utils.js";
 import {sections} from "../services-data.js";
+import Card from "../../../components/Card/index.jsx";
```
Comment (vprince001) [inline src/pages/Services/Cards/index.jsx:4]: remove index.jsx as it is not needed.

File: src/components/Card/index.jsx (hunk 1 above)
Comment (harshitverma2103) [reply src/components/Card/index.jsx]: Resolved

File: src/components/Card/index.jsx (hunk 2 above)
Comment (harshitverma2103) [reply src/components/Card/index.jsx]: Resolved

File: src/components/WhyUsCards/index.jsx (hunk 3 above)
Comment (harshitverma2103) [reply src/components/WhyUsCards/index.jsx]: Resolved

File: src/components/WhyUsCards/whyUs-data.js (hunk 4 above)
Comment (harshitverma2103) [reply src/components/WhyUsCards/whyUs-data.js]: Resolved

File: src/components/WhyUsCards/whyUs-data.js [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    title: "Our Mission",
+    content:
+      "To revolutionize the IT industry by pushing the boundaries of innovation. We strive to be at the forefront of technological advancement, continuously pioneering new solutions that empower businesses and individuals to thrive in a rapidly evolving digital landscape.",
+  },
+  {
+    icon: tech,
+    title: "Technologies",
+    content:
+      "We offer a comprehensive range of IT services. Our services are designed to help small businesses thrive in today's digital world.",
+    url: "Learn More",
```
[the remark this answers is not in the document]
Comment (harshitverma2103) [reply src/components/WhyUsCards/whyUs-data.js]: Resolved

File: src/pages/Services/Cards/index.jsx (hunk 5 above)
Comment (harshitverma2103) [reply src/pages/Services/Cards/index.jsx:4]: Resolved

File: src/components/WhyUsCards/index.jsx [hunk 7]
```diff
@@ -1,50 +1,37 @@
 import React from "react";
+import { useNavigate } from "react-router-dom";
+import { SERVICES_PAGE_PATH } from "../../constants.js";
+import Card from "../Card";
+import { data } from "./whyUsCardData.js";
```
Comment (vprince001) [inline src/components/WhyUsCards/index.jsx:5]: Fix these imports

File: src/components/WhyUsCards/index.jsx [hunk 8]
```diff
@@ -1,50 +1,37 @@
...
+      {data.map((cardData, index) => {
+        const { icon, title, content, linkLable } = cardData;
+        return (
+          <div key={index} className="why-us-card">
+            <Card
+              icon={icon}
+              title={title}
+              content={content}
+              linkLable={linkLable}
+              onLinkClick={() => handleOnClick(linkLable)}
```
Comment (vprince001) [inline src/components/WhyUsCards/index.jsx:28]: we don't need handleOnClick here. We can directly use navigate.

File: src/components/WhyUsCards/index.jsx [hunk 9]
```diff
@@ -1,50 +1,37 @@
...
+    <div className="why-us-card-container">
+      {data.map((cardData, index) => {
+        const { icon, title, content, linkLable } = cardData;
+        return (
+          <div key={index} className="why-us-card">
+            <Card
+              icon={icon}
+              title={title}
+              content={content}
+              linkLable={linkLable}
```
Comment (vprince001) [inline src/components/WhyUsCards/index.jsx:27]: It should be linkLabel

File: src/pages/Services/Cards/index.jsx [hunk 10]
```diff
@@ -1,24 +1,9 @@
 import React, {useEffect, useState} from "react";
 import {scrollToElement} from "../../../utils.js";
 import {sections} from "../services-data.js";
+import Card from "../../../components/Card";
```
Comment (vprince001) [inline src/pages/Services/Cards/index.jsx:4]: Fix imports order -
Library
Component
Utils
Data
Style

File: src/components/WhyUsCards/whyUsCardData.js [hunk 11]
```diff
@@ -0,0 +1,25 @@
+import group from "../../assets/icons/group.png";
```
Comment (vprince001) [inline src/components/WhyUsCards/whyUsCardData.js:1]: file name should be whyUsCardsData.js

File: src/components/WhyUsCards/index.jsx (hunk 3 above)
Comment (vprince001) [reply src/components/WhyUsCards/index.jsx]: file name should be whyUsCardsData.js

File: src/components/WhyUsCards/whyUs-data.js (hunk 4 above)
Comment (vprince001) [reply src/components/WhyUsCards/whyUs-data.js]: I don't see these changes.

File: src/components/WhyUsCards/whyUsCardData.js [hunk 12]
```diff
@@ -0,0 +1,25 @@
+import group from "../../assets/icons/group.png";
+import mortarboard from "../../assets/icons/mortarboard.png";
+import tech from "../../assets/icons/coding.png";
+
+export const data = [
```
Comment (vprince001) [inline src/components/WhyUsCards/whyUsCardData.js:5]: use default export also name it as whyUsCardsData

File: src/components/Card/style.css [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+  text-align: center;
+  color: #393939;
+}
+
+.card-content {
+  padding: 0 20px 20px;
+  text-align: center;
+}
+
+.card-linklable {
```
Comment (vprince001) [inline src/components/Card/style.css:45]: fix this class name

File: src/components/WhyUsCards/index.jsx (hunk 3 above)
Comment (harshitverma2103) [reply src/components/WhyUsCards/index.jsx]: Done

File: src/components/WhyUsCards/whyUsCardData.js (hunk 12 above)
Comment (harshitverma2103) [reply src/components/WhyUsCards/whyUsCardData.js]: Done

File: src/components/Card/style.css (hunk 13 above)
Comment (harshitverma2103) [reply src/components/Card/style.css]: Done

File: src/pages/Services/Cards/index.jsx (hunk 10 above)
Comment (harshitverma2103) [reply src/pages/Services/Cards/index.jsx]: Done

File: src/components/Card/style.css (hunk 13 above)
Comment (harshitverma2103) [reply src/components/Card/style.css]: Done

**Pull request merged**